src/LanguageContext.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
```

Comment:
Discussion: I wonder if we could have this context in `src/translation` folder and move the `useTranslation` hook as a part of this module. With this we would co-locate everything relevant for translations to that particular module. Also we could do `import { useTranslation } from 'src/translation'` at some point.

Reply:
Agreed, done :tada: